modules/ROOT/pages/project-structure.adoc

```diff
@@ -1,89 +1,148 @@
 = Project structure
 :description: This page defines what the elements are within a Bonita Automation project, as well as how some of them are structured to create a Living Application.
 
-This page defines what the elements are within a Bonita Automation project as well as how some of the are structured to create a Living Application.
+This page defines the elements used to compose a Bonita Automation project.
 
-An automation project can be of different kinds. 
-We like to promote the Living Applications based on process execution, because it takes advantage of the full potential of Bonita. +
-But it can also by about process execution using the Bonita provided applications, with no need for a custom application. +
-Or about process execution with no application, when the process is about automated tasks (and robots) only.
+== Bonita Automation project elements
```

Comment:
The above paragraph is pretty important to the Product Management team as well as for the Pre-Sales team.
It will also be described in the Training.